### ISSUE TITLE
SciPy-bundle-2021.10-foss-2021b fails to build on POWER9

### ISSUE THREAD

**Issue by golivag** [easybuilders/easybuild-easyconfigs#15987]
SciPy bundle fails to build on power 9 (RHEL8).
Here is the the relevant part of the log file:

```
linalg/tests/test_linalg.py ...............Fatal Python error: Segmentation fault

Current thread 0x00007fffa12e8280 (most recent call first):
  File "/tmp/eb-vttfywrm/tmpz41obj6e/lib/python3.9/site-packages/numpy/linalg/linalg.py", line 1323 in eig
  File "<__array_function__ internals>", line 5 in eig
  File "/tmp/eb-vttfywrm/tmpz41obj6e/lib/python3.9/site-packages/numpy/linalg/tests/test_linalg.py", line 558 in do
  File "/tmp/eb-vttfywrm/tmpz41obj6e/lib/python3.9/site-packages/numpy/linalg/tests/test_linalg.py", line 85 in check
  File "/tmp/eb-vttfywrm/tmpz41obj6e/lib/python3.9/site-packages/numpy/linalg/tests/test_linalg.py", line 350 in check_cases
  File "/tmp/eb-vttfywrm/tmpz41obj6e/lib/python3.9/site-packages/numpy/linalg/tests/test_linalg.py", line 360 in test_sq_cases
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/_pytest/python.py", line 183 in pytest_pyfunc_call
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/pluggy/callers.py", line 187 in _multicall
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/pluggy/manager.py", line 84 in <lambda>
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/pluggy/manager.py", line 93 in _hookexec
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/pluggy/hooks.py", line 286 in __call__
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/_pytest/python.py", line 1641 in runtest
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/_pytest/runner.py", line 162 in pytest_runtest_call
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/pluggy/callers.py", line 187 in _multicall
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/pluggy/manager.py", line 84 in <lambda>
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/pluggy/manager.py", line 93 in _hookexec
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/pluggy/hooks.py", line 286 in __call__
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/_pytest/runner.py", line 255 in <lambda>
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/_pytest/runner.py", line 311 in from_call
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/_pytest/runner.py", line 254 in call_runtest_hook
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/_pytest/runner.py", line 215 in call_and_report
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/_pytest/runner.py", line 126 in runtestprotocol
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/_pytest/runner.py", line 109 in pytest_runtest_protocol
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/pluggy/callers.py", line 187 in _multicall
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/pluggy/manager.py", line 84 in <lambda>
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/pluggy/manager.py", line 93 in _hookexec
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/pluggy/hooks.py", line 286 in __call__
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/_pytest/main.py", line 348 in pytest_runtestloop
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/pluggy/callers.py", line 187 in _multicall
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/pluggy/manager.py", line 84 in <lambda>
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/pluggy/manager.py", line 93 in _hookexec
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/pluggy/hooks.py", line 286 in __call__
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/_pytest/main.py", line 323 in _main
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/_pytest/main.py", line 269 in wrap_session
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/_pytest/main.py", line 316 in pytest_cmdline_main
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/pluggy/callers.py", line 187 in _multicall
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/pluggy/manager.py", line 84 in <lambda>
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/pluggy/manager.py", line 93 in _hookexec
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/pluggy/hooks.py", line 286 in __call__
  File "/opt/software/easybuild/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/site-packages/_pytest/config/__init__.py", line 162 in main
  File "/tmp/eb-vttfywrm/tmpz41obj6e/lib/python3.9/site-packages/numpy/_pytesttester.py", line 197 in __call__
  File "<string>", line 1 in <module>
 (at easybuild/tools/run.py:618 in parse_cmd_output)
```

Any help appreciated.

**Comment by boegel** [comment]
@golivag This is due to a miscompilation of OpenBLAS on POWER that causes trouble with the FlexiBLAS component of `foss/2021a`. A fix for this is available in #15885 (which is still being evaluated).

Can you try (re)building OpenBLAS from that PR (using `eb --from-pr 15885 OpenBLAS-0.3.18-GCC-11.2.0.eb`, and check if that resolves the problem for you too?

cc @Flamefire

**Comment by golivag** [comment]
Yes it does. 
Thank you very much for your help!
